utils.py

```diff
@@ -747,6 +747,32 @@ def get_hashable_value(value):
         return value
 
 
+def compress_to_zlib(data):
+    """Compress the data to zlib format for efficient storage and communication.
+
+    Args:
+        data: str. Data to be copressed.
+
+    Returns:
+        Compressed data string.
+    """
+    import zlib
```

Comment:
This is weird ... why not import at the top of the file, like other stdlibs? Ditto below.

Reply:
I think I just copied it here from python_utils without realizing the difference in the conventions used in both files.
Done